### PR TITLE
adding a security monitoring section to docs

### PR DESCRIPTION
**Pull request by cswatt**
### What does this PR do?
adds a Security Monitoring section

### Preview link

https://docs-staging.datadoghq.com/cswatt/security/security_monitoring
https://docs-staging.datadoghq.com/cswatt/security/security_monitoring/explorer
https://docs-staging.datadoghq.com/cswatt/security/security_monitoring/detection_rules

**note** that certain adblockers will take issue with the preview name, because “security/security” triggers something on their blacklist. if you’re having trouble viewing this, disable your adblocker or view in incognito. this issue will not persist outside of staging (i just named my branch badly)

### CODE REVIEW

File: content/en/security_monitoring/_index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Security Monitoring
+kind: documentation
+---
+
+## Overview
+
+Datadog Security Monitoring enbales you to detect threats in real time and investigate security alerts across your infrastructure metrics, distributed traces, and logs.
```
Comment (sarina-dd) [inline content/en/security_monitoring/_index.md]:
```suggestion
Datadog Security Monitoring enables you to detect threats in real time and investigate security alerts across your infrastructure metrics, distributed traces, and logs.
```

File: content/en/security_monitoring/detection_rules.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+A signal will “close” regardless of the query being matched once this time threshold is met. This time is calculated from the time the signal is generated. 
+
+Additional cases can be added by clicking the **Add Case** button. 
+
+### Security Signal Message
+
+{{< img src="security_monitoring/detection_rules/say_whats_happening.png" alt="Say what's happening" responsive="true">}}
+
+The notification box has the same Markdown and **Preview** features as those of [monitor notifications][2]
```
Comment (sarina-dd) [inline content/en/security_monitoring/detection_rules.md]:
```suggestion
The notification box has the same Markdown and **Preview** features as those of [monitor notifications][2].
```

File: content/en/security_monitoring/detection_rules.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+### Define the rule cases
+
+{{< img src="security_monitoring/detection_rules/set_rule_cases.png" alt="Set rule cases" responsive="true">}}
+
+#### Rule case trigger and name
+
+Rule cases, such as `a > 3`, are evaluated as case statements. Thus, the first case to match generates the signal. Click and drag your rule cases to manipulate their ordering.
+
+A rule case contains logical operations (`>, <, >=, <=, &&, ||, !`) to determine if a signal should be generated based on the event counts in the previously defined queries. The ASCII lowercase [query labels](#define-the-search-query) are referenced in this section.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/detection_rules.md]: `>, <, >=, <=, &&, ||, !` should be `>,  >=, &&, ||`

Also, there should be there should be some additional text related to rule case that explains the following:

> The query label must be before the operator. Example: `a > 3`. The following is NOT allowed: `3 > a`

Comment (jmassey-datadog) [inline content/en/security_monitoring/detection_rules.md]:
```suggestion
A rule case contains logical operations (`>, >=, &&, ||`) to determine if a signal should be generated based on the event counts in the previously defined queries. The ASCII lowercase [query labels](#define-the-search-query) are referenced in this section. 
```

File: content/en/security_monitoring/detection_rules.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+kind: documentation
+further_reading:
+- link: "/security_monitoring/explorer"
+  tag: "Documentation"
+  text: "Search through all of your security signals and perform Security Analytics"
+---
+
+## Overview
+
+Use Detection Rules to define conditional logic that is applied to your entire event stream. When at least one case defined in a Detection Rule is matched over a given period of time, Datadog generates a Security Signal.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/detection_rules.md]:
```suggestion
Use Detection Rules to define conditional logic that is applied to all ingested logs. When at least one case defined in a Detection Rule is matched over a given period of time, Datadog generates a Security Signal.
```

File: content/en/security_monitoring/detection_rules.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+### Security Signal Message
+
+{{< img src="security_monitoring/detection_rules/say_whats_happening.png" alt="Say what's happening" responsive="true">}}
+
+The notification box has the same Markdown and **Preview** features as those of [monitor notifications][2]
+
+The **Rule name** section allows you to configure the rule name that appears in the rules list view, as well as the title of the Security Signal when it is generated.
+
+You can tag your signals with different tags, for example `security:anomaly` or `security:threat-intel`.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/detection_rules.md]:
```suggestion
You can tag your signals with different tags, for example `security:attack` or `technique:T1110-brute-force`.

Note: the tag `security` is special. This tag is used to classify the security signal. The recommended options are: `attack`, `threat-intel`, `compliance`, `anomaly`, and `data-leak`.
```

File: content/en/security_monitoring/detection_rules.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+## Manage Detection Rules
+
+The [Security Configuration Detection Rules][3] page lets you search of all Detection Rules so you can delete or disable the rules. You can also clone or fully edit any individual rule in the search results.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/detection_rules.md]:
```suggestion
The [Security Configuration Detection Rules][3] page lets you search of all Detection Rules. You can enable/disable, edit, delete, clone, or view signals generated by any of these rules. You can also create a new rule from scratch from this view.
```

File: content/en/security_monitoring/detection_rules.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+{{< img src="security_monitoring/detection_rules/detection_rules.png" alt="Share Signal" responsive="true" style="width:50%;">}}
+
+### Finding rules
+
+The free text search lets you query Detection Rules by text in the rule name or query. Query results update in real-time as you edit the query—there is no “Search” button to click.  
+
+### Detection Rules Table
+The Detection Rules are displayed in the Detection Rules table.
+
+Configure the logs table content according to your needs and preferences with the **Options** button. Only date created and rule ID are available for additional columns.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/detection_rules.md]:
```suggestion
Configure the Detection Rules table content according to your needs and preferences with the **Options** button. Only date created and rule ID are available for additional columns.
```

File: content/en/security_monitoring/explorer.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+In this view, you can:
+
+* [Build a context](#context) to explore your Security Signals.
+* [Visualize your logs](#visualization) as a filtered Security Signals table or Security Signals Analytics.
+* [Share the content](#share-views) of your Security Signal Explorer view to another page within or outside of Datadog.
+
+## Context
+
+Build up a context to explore your signals in your Security Signals Explorer view. First, select the proper time range. Then, use the search bar to filter your signals.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/explorer.md]:
```suggestion
Build up a context to explore your signals in your Security Signals Explorer view. First, select the proper time range. Then, use the [search](#search) bar to filter your signals.
```

File: content/en/security_monitoring/explorer.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
...
+{{< img src="security_monitoring/explorer/search_your_signals.png" alt="Search your signals" responsive="true" >}}
+
+### Share views
+
+Export your current security signal visualization with the *share* functionality:
+
+{{< img src="security_monitoring/explorer/share_security_signals_view.png" alt="Share Security Signals view" responsive="true" style="width:60%;">}}
+
+
+Use the *share* button to send your current log explorer view to a team member.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/explorer.md]:
```suggestion
Use the *share* button to send your current Security Signal explorer view to a team member.
```

File: content/en/security_monitoring/explorer.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+
+
+## Visualization
+
+Switch between the Security Signals Search and the Security Signals Analytics modes by clicking on the *Signal Mode* button in the upper left corner of the page:
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/explorer.md]:
```suggestion
Switch between the Security Signals Table and the Security Signals Analytics modes by clicking on the *Signal Mode* button in the upper left corner of the page:
```

File: content/en/security_monitoring/explorer.md [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+
+{{< img src="security_monitoring/explorer/visualization_switch.png" alt="Visualization switch" responsive="true" style="width:50%;">}}
+
+
+The Security Signals Search is the list of signals that match the selected context. A context is defined by a [search bar](#search) filter and a [time range](#time-range).
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/explorer.md]:
```suggestion
The Security Signals Table displays the list of signals that match the selected context. A context is defined by a [search bar](#search) filter and a [time range](#time-range).
```

File: content/en/security_monitoring/explorer.md [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+
+
+### Security Signals Table
+
+The Security Signals Search is displayed in the Security Signals table.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/explorer.md]:
```suggestion
The Security Signals search results are displayed in the Security Signals Table.
```

File: content/en/security_monitoring/explorer.md [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+
+Security Signal results are sorted by date—the most recent on top by default. You can also inverse-sort by date, with the least recent (within the limits of the time range) on top.
+
+{{< img src="security_monitoring/explorer/configure_display_table.png" alt="Configure Display Table" responsive="true" style="width:50%;">}}
+
+### Security Signal Panel
+
+Click on any Security Signal to open the Security Signal Panel and see more details about it: signal creation date, first seen date, last seen date, classification, tags, security signal message, and the samples that triggered the Security Signal.
+ 
+Use the **Share** button to share the log opened in side panel to other contexts.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/explorer.md]:
```suggestion
Use the **Share** button to share the Security Signal opened in side panel to a team member or any other Datadog notification integration. 
```

File: content/en/security_monitoring/explorer.md [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+
+From the Security Signals Explorer, you can triage security signals.
+
+{{< img src="security_monitoring/explorer/security_signals_explorer_view.png" alt="Security Signals Explorer view" responsive="true" >}}
+
+
+In this view, you can:
+
+* [Build a context](#context) to explore your Security Signals.
+* [Visualize your logs](#visualization) as a filtered Security Signals table or Security Signals Analytics.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/explorer.md]:
```suggestion
* [Visualize your Security Signals](#visualization) as a filtered Security Signals table or Security Signals Analytics.
```

File: content/en/security_monitoring/explorer.md [hunk 15]
```diff
@@ -0,0 +1,114 @@
...
+From the Security Signals Explorer, you can triage security signals.
+
+{{< img src="security_monitoring/explorer/security_signals_explorer_view.png" alt="Security Signals Explorer view" responsive="true" >}}
+
+
+In this view, you can:
+
+* [Build a context](#context) to explore your Security Signals.
+* [Visualize your Security Signals](#visualization) as a filtered Security Signals table or Security Signals Analytics.
+* [Share the content](#share-views) of your Security Signal Explorer view to another page within or outside of Datadog.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/explorer.md]:
```suggestion
* Inspect a security signal in the [Security Signal Panel](#security-signal-panel)
```

File: content/en/security_monitoring/explorer.md [hunk 16]
```diff
@@ -0,0 +1,114 @@
...
+### Security Signal Panel
+
+Click on any Security Signal to open the Security Signal Panel and see more details about it: signal creation date, first seen date, last seen date, classification, tags, security signal message, and the samples that triggered the Security Signal.
+
+{{< img src="security_monitoring/explorer/panel.png" alt="Security Signal Panel" responsive="true" style="width:80%;">}}
+
+#### Explanation of the above example
+
+- **Severity**: The `HIGH` indicator in the top left denotes the severity of the highest case matched.
+- **Signal generation time**: Located to the right of the severity indicator, this denotes when any of the cases first matched. It is not updated.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/explorer.md]:
```suggestion
- **Signal generation time**: Located to the right of the severity indicator, this denotes when any of the cases first matched. It is never updated.
```

File: content/en/security_monitoring/explorer.md [hunk 17]
```diff
@@ -0,0 +1,114 @@
...
+
+
+The Security Signals Table displays the list of signals that match the selected context. A context is defined by a [search bar](#search) filter and a [time range](#time-range).
+
+
+### Security Signals Table
+
+The Security Signals search results are displayed in the Security Signals Table.
+
+Configure the content of your Security Signals Table according to your needs and preferences with the **Options** button. Only faceted attributes are available for columns.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/explorer.md]:
```suggestion
Configure the content of your Security Signals Table according to your needs and preferences with the **Options** button. All signal attributes are available for columns.
```

File: content/en/security_monitoring/_index.md [hunk 18]
```diff
@@ -0,0 +1,32 @@
...
+
+Datadog detects threats out-of-the-box and can notify your team of security issues by email, Slack, Jira, PagerDuty, or a webhook. 
+
+{{< img src="security_monitoring/takeover_ex.png" alt="Slack Example" responsive="true" style="width:75%;">}}
+
+Datadog detects many different sorts of threats to your application or infrastructure. A few examples include a targeted attack, an IP communicating with your systems which matches a threat intel list, or an insecure configuration. These threats are surfaced in Datadog as Security Signals and can be correlated and triaged in the [Security Explorer][1]. 
+
+{{< img src="security_monitoring/explorer.png" alt="Security Signals Explorer" responsive="true" >}}
+
+Security Signals are generated by Datadog Security Monitoring with [Detection Rules][2]. Detection Rules detect threats across different sources and are available out of the box for immediate use. Any of the provided detection rules can be cloned to change the configuration. A new rule can also be created from scratch.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/_index.md]: can we link https://app.datadoghq.com/security/configuration/rules/new on the words `new rule`?

File: content/en/security_monitoring/detection_rules.md [hunk 19]
```diff
@@ -0,0 +1,137 @@
...
+
+The Detection Rules join the logs together using a group by value. The group by values are typically entities (e.g. IP address, user, etc), but can be any attribute. 
+
+{{< img src="security_monitoring/detection_rules/group_by.png" alt="Group by" responsive="true" >}}
+
+The Detection Rule cases join these queries together based on their group by value. The group by attribute is typically the same attribute because the value must be the same for the case to be met. If a group by value doesn’t exist, the case will never be met. A Security Signal is generated for each unique group by value when a case is matched.
+
+{{< img src="security_monitoring/detection_rules/set_rule_cases2.png" alt="Set rule cases" responsive="true" >}}
+
+In this example, when 5 failed logins and a successful login exist for the same `@usr.name`, the first case is matched and a Security Signal is generated.
```
Comment (jmassey-datadog) [inline content/en/security_monitoring/detection_rules.md]:
```suggestion
In this example, when greater than 5 failed logins and a successful login exist for the same `@usr.name`, the first case is matched and a Security Signal is generated. 
```

File: content/en/security_monitoring/_index.md (hunk 18 above)
Comment (cswatt) [reply content/en/security_monitoring/_index.md]: yup, done

**Pull request merged**